### PR TITLE
Fix the outline colour for the wp-checkout component

### PR DESCRIPTION
**Pull request by fditrapani**
#### Changes proposed in this Pull Request

This PR changes the default outline colour.

Before:
image

After:
image

#### Testing instructions

Follow instructions here: https://github.com/Automattic/wp-calypso/pull/36720

Fixes #

### CODE REVIEW

File: packages/wp-checkout/public/index.html [hunk 1]
```diff
@@ -13,6 +13,10 @@
 			-moz-osx-font-smoothing: grayscale;
 			background: #F3F4F5;
 		}
+
+		*:focus {
```
Comment (sirbrillig) [inline packages/wp-checkout/public/index.html]: These styles will only affect this page, which is just a demo for a host page. Is this a style rule we will want for all our hosts? If so, we should include it somewhere in the files in `src/components`.

Comment (sirbrillig) [reply packages/wp-checkout/public/index.html]: note to self: maybe I should rename this directory from `public` to `demo` to make it more obvious what it does!

Comment (fditrapani) [reply packages/wp-checkout/public/index.html]: oh right. Good point! I'll move it to the parent container.

**Review by sirbrillig — APPROVED** [review]
Nice! Now, I wonder what happens if we just merge this in github?

**Comment by sirbrillig** [comment]
Merged manually into #36720 to avoid a squash.

**Pull request closed without merging**